### ISSUE TITLE
Cannot validate email anonymously

### ISSUE THREAD

**Issue by lobodol** [symfony/maker-bundle#786]
Hello everyone,

I'm trying to add an email verification in my project without having to be logged in. Thus, I used this bundle & [SymfonyCastsVerifyEmailBundle](https://github.com/SymfonyCasts/verify-email-bundle) and ran the follwoing command:

```bash
bin/console make:registration-form
```

# Actual result
I saw in [this PR](https://github.com/symfony/maker-bundle/pull/776/files) that when I answer NO to the question "Do you want to automatically authenticate the user after registration?" the generated controller should **not** deny access unless granted.
But, here is a part of the generated controller I got:

![Make command](https://user-images.githubusercontent.com/7249581/104576041-fe171080-5657-11eb-855c-3000cfc87c8c.png)

```php
    /**
     * @Route("/verify/email", name="app_verify_email")
     */
    public function verifyUserEmail(Request $request): Response
    {
        $this->denyAccessUnlessGranted('IS_AUTHENTICATED_FULLY');

        // validate email confirmation link, sets User::isVerified=true and persists
        try {
            $this->emailVerifier->handleEmailConfirmation($request, $this->getUser());
        } catch (VerifyEmailExceptionInterface $exception) {
            $this->addFlash('verify_email_error', $exception->getReason());

            return $this->redirectToRoute('app_register');
        }

        // @TODO Change the redirect on success and handle or remove the flash message in your templates
        $this->addFlash('success', 'Your email address has been verified.');

        return $this->redirectToRoute('app_register');
    }
```

This is the same thing for all other generated files.

# Expected result
I would expect the controller to be generated as the following:

```php
public function verifyUserEmail(Request $request, UserRepository $repository): Response
    {
        $id = $request->get('id');

        if (null === $id) {
            return $this->redirectToRoute('app_register');
        }
        
        $user = $repository->find($id);

        if (null === $user) {
            return $this->redirectToRoute('app_register');
        }
        
        // validate email confirmation link, sets User::isVerified=true and persists
        try {
            $this->emailVerifier->handleEmailConfirmation($request, $user);
        } catch (VerifyEmailExceptionInterface $exception) {
            $this->addFlash('verify_email_error', $exception->getReason());

            return $this->redirectToRoute('app_register');
        }

        // @TODO Change the redirect on success and handle or remove the flash message in your templates
        $this->addFlash('success', 'Your email address has been verified.');

        return $this->redirectToRoute('app_register');
    }
```

# Dependencies versions
* Symfony: v4.4.18
* MakerBundle: v1.26.1
* symfonycasts/verify-email-bundle: v1.1.1

Am I using it wrong?

Regards

**Comment by lobodol** [comment]
Oh, my bad. I update makerbundle to v1.27 and the generated controller now allows anonymous email verification.

**But**, the generated code is not functional. Indeed, the variable `$user` is not affected:

```php
    public function verifyUserEmail(Request $request, UserRepository $userRepository): Response
    {
        $id = $request->get('id');

        if (null === $id) {
            return $this->redirectToRoute('app_register');
        }

        $userRepository->find($id);

        if (null === $user) {
            return $this->redirectToRoute('app_register');
        }
        //...
```

**Comment by lobodol** [comment]
Bug fix has been shipped in [release 1.28.0](https://github.com/symfony/maker-bundle/releases/tag/v1.28.0).
